### DISCUSSION TITLE
Cannot detect CPU on HP Elitedesk with Ryzen 2400GE

### DISCUSSION THREAD

**Discussion by Karjack182** [DavidS95/Smokeless_UMAF#52] [Q&A]
Just tried this software, and all the devices that this detects is the network device.
Cant find the cpu to do any editing :(

Does anyone have any ideas, or different software to recommend? The HP Bios is extremely restrictive

**Comment by EirFun** [comment]
Same here, only the network device is detected

**Comment by Grabulosaure** [comment]
Same on EliteDesk 805 DM G8 with R5pro 5650GE. Only network device

**Comment by angelokimhui** [comment]
I have same problem, I don't see any options at all. I instead have Elitebook 845 G7. It seems like HP has different offset than the common ones. I just instead want to know how to overcome it. Knowing that this software is closed source. I'm not sure as well what to do.

**Comment by killerharvey** [comment]
I have exactly the same issue on my HP Elitedesk 705 G5 with an Amd 3400GE (Picasso). I tried rolling back to older bios but the issue persists. I was really looking forward to maxing out the vram on the Vega11
